learn/getting_started/customizing_relevancy.md

```diff
@@ -126,7 +126,7 @@ The `synonyms` array lets you associate certain words in your dataset, telling M
 
 <CodeSamples id= "getting_started_synonyms" />
 
-This will set `winnie` and `piglet` as synonyms; searching for either word will show the same results.
+This will set `winnie` and `piglet` as synonyms. Searching for either won't always return the same results due to factors like allowed typos and splitting the word.
```

Comment:
I think this addition is great, but requires a bit more work.

1. What do we mean with "allowed typos"?  Perhaps just "typos" is better because users are not lead to wonder about allowed vs disallowed typos?

2. I don't understand what we mean by "splitting the word." Can you clarify it?